tools/src/main/java/org/apache/kafka/tools/consumer/group/ConsumerGroupCommand.java

```diff
@@ -820,7 +869,11 @@ TreeMap<String, Entry<Optional<GroupState>, Optional<Collection<MemberAssignment
                         consumer.clientId(),
                         consumer.groupInstanceId().orElse(""),
                         consumer.assignment().topicPartitions().size(),
-                        new ArrayList<>(verbose ? consumer.assignment().topicPartitions() : Collections.emptySet())
+                        new ArrayList<>(verbose ? consumer.assignment().topicPartitions() : Collections.emptySet()),
```

Comment:
Do we really need to take `verbose` into consideration here? It is a bit weird to consider it here but not for the others.